rep-2004.rst

```diff
@@ -57,239 +62,388 @@ Quality Level 1
-    * [ROS Core]:
+      #. *Must have performance tests (exceptions allowed if they don't make sense to have)*
+      #. *Must have a performance regression policy (i.e. blocking either changes or releases on unexpected performance regressions)*
 
-      * May have performance tests, strongly recommended, but for some packages it doesn't make sense
-      * If there are performance tests, must choose to either check each change or before each release or both
-      * If there are performance tests, must require justification for merging a change or making a release that lowers performance
+   #. Linters and Static Analysis
 
-  * Linters and Static Analysis
+      #. *Must have a code style and enforce it*
+      #. *Must use static analysis tools where applicable*
 
-    * Must have a code style and enforce it.
-    * Must use static analysis tools where applicable.
-    * [ROS Core]:
+#. **Dependencies:**
 
-      * Must use ROS code style and use linters from ``ament_lint_common`` to enforce it
-      * Must use all linters/static analysis that are part of ``ament_lint_common``
+   #. Must not have direct runtime "ROS" dependencies which are not at the same level as the package in question ('Level N'), but...
+   #. May have optional direct runtime "ROS" dependencies which are not 'Level N', e.g. tracing or debugging features that can be disabled
+   #. Must have justification for why each direct runtime "non-ROS" dependency is equivalent to a 'Level N' package in terms of quality
 
-* Dependencies:
+#. **Platform Support:**
 
-  * Must not have direct runtime "ROS" dependencies which are not 'Level 1' dependencies, but...
-  * May have optional direct runtime "ROS" dependencies which are not 'Level 1', e.g. tracing or debugging features that can be disabled
-  * Must have justification for why each direct runtime "non-ROS" dependency is equivalent to a 'Level 1' package in terms of quality
-
-* Platform Support:
-
-  * Must support all tier 1 platforms for ROS 2, as defined in `REP-2000 <https://www.ros.org/reps/rep-2000.html#support-tiers>`_
+   #. Must support all tier 1 platforms for ROS 2, as defined in `REP-2000 <https://www.ros.org/reps/rep-2000.html#support-tiers>`_
 
 If the above points are satisfied then a package can be considered 'Level 1'.
-Below are some details on the above points.
+Refer to the detailed description of the requirements below the chart for more information.
+
+Quality Level Comparison Chart
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The chart below compares Quality Levels 1 through 5 relative to the 'Level 1' requirements' numbering scheme above.
+
+✓ = required
+
+● = recommended
+
+.. list-table:: Quality Levels
+    :widths: 5 10 10 10 10 10
+    :header-rows: 1
+    :stub-columns: 1
+    :align: left
+
+
+    * -
+      - Level 1
+      - Level 2
+      - Level 3
+      - Level 4
+      - Level 5
+    * - 1.i
```

Comment:
It would be great adding a column with a brief description of the item after the roman numerals, e.g 
`" 1.i.  |  version policy "`
`" 1.ii. |  stable version "`
So it's easier to read the chart on its own, without scrolling back to the top to remember

Reply:
Yes, we discussed this back and forth a bit, but we had issues with the formatting. I tend to agree that having only the number references makes it hard to read. A compromise might be to have links for the numbers to the sections to which they correspond.